src/main/java/com/seatapp/controllers/LoginController.java

```diff
@@ -0,0 +1,122 @@
+package com.seatapp.controllers;
+
+import com.seatapp.services.usermanagement.UserService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.security.oauth2.core.user.OAuth2User;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletResponse;
+import java.net.URI;
+
+@CrossOrigin(origins = "http://localhost:19006", allowCredentials = "true")
+@RestController
+@RequestMapping("api")
+public class LoginController {
+    /**
+     * The logger of the login controller.
+     */
+    private static final Logger LOGGER =
+            LoggerFactory.getLogger(LoginController.class);
+
+    /**
+     * Represents the user service.
+     */
+    private final UserService userService;
+
+    /**
+     * Creates a login controller.
+     *
+     * @param userService the user service to manage the users.
+     */
+    @Autowired
+    public LoginController(final UserService userService) {
+        this.userService = userService;
+    }
+
+    /**
+     * Check if the user can reach this api call.
+     *
+     * @param token the JWT token of the user that reached the url
+     * @return a http response OK
+     */
+    @GetMapping("healthcheck")
+    public ResponseEntity<Void> healthCheck(
+            final UsernamePasswordAuthenticationToken token) {
+        if (LOGGER.isDebugEnabled()) {
```

Comment:
Are these loggers usefull? At the moment they don't log anything.

Reply:
They can be used to find bugs and see if the flow of the application is correct.